front/app/routes/buildbots.js

```diff
@@ -9,14 +9,14 @@ function fixTimestamps(data) {
 
 export default Ember.Route.extend({
   model: function() {
-    var summary_url = ENV.apiBaseURL + 'buildbots';
+    const summary_url = ENV.apiBaseURL + 'buildbots';
     return Ember.$.getJSON(summary_url)
       .then(metrics => {
 
-        var win_buildbot_times = [];
-        var mac_buildbot_times = [];
-        var linux_buildbot_times = [];
-        var misc_buildbot_times = [];
+        const win_buildbot_times = [];
```

Comment:
Yeah, `const` is just a constant *binding*. Unfortunately, there isn't anything to indicate constant *value* for e.g. objects or arrays (though you can use `Object.freeze()` to make a given object instance constant now).